core/core/src/sf_core/json_schema_validator.rs

```diff
@@ -183,5 +183,17 @@ mod test {
             .unwrap(),
         );
         assert!(result.is_err());
+
+        let result = security_validator.validate(
+            &HostValue::deserialize(&json!({
+                "security_config": {
+                    "partial_basic": {
```

Comment:
issue: I think this doesn't test what we want it to - the `security_config` wrapper should no be there:
```
{
  "partial_basic": { .. }
}
```